src/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Responsive Mockup</title>
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+  </head>
+
+  <body>
+
+		<form>
+			<h1>Login</h1>
+		<!-- email input -->
+			<label for="email">Email</label>
+					<br>
+			<input id="email" class="full-width" type="text" placeholder="you@mail.com"></input>
```

Comment:
Instead of doing JS validation, for this exercise you can just have HTML Validations (e.g. `required`)